appeals/web/src/server/appeals/personal-list/personal-list.service.js

```diff
@@ -4,12 +4,24 @@ import { paginationDefaultSettings } from '../appeal.constants.js';
 
 /**
  * @param {import('got').Got} apiClient
+ * @param {string|undefined} appealStatusFilter
  * @param {number} pageNumber
  * @param {number} pageSize
  * @returns {Promise<AppealList>}
  */
 export const getAppealsAssignedToCurrentUser = (
 	apiClient,
+	appealStatusFilter,
 	pageNumber = paginationDefaultSettings.firstPageNumber,
 	pageSize = paginationDefaultSettings.pageSize
-) => apiClient.get(`appeals/my-appeals?pageNumber=${pageNumber}&pageSize=${pageSize}`).json();
+) => {
+	let urlAppendix = '';
```

Comment:
Again super minor, but this variable name could be a little clearer - maybe `filterQuery` or similar?

Reply:
It really is just an url append - i think it should stay this way as more updates are anticipated in near future